Documentation/concepts/kubernetes/requirements.rst

```diff
@@ -51,8 +51,11 @@ The :ref:`k8s_install_etcd_operator` relies on the etcd-operator to manage an
 etcd cluster. In order for the etcd cluster to be available, the Cilium pod is
 being run with ``dnsPolicy: ClusterFirstWithHostNet`` in order for Cilium to be
```

Comment:
Would it make sense to add a link to the description of the policies?
e.g., https://pkg.go.dev/k8s.io/api@v0.20.2/core/v1#DNSPolicy
(not sure if there is a more stable link we can use...)